lib/SIL/TypeLowering.cpp

```diff
@@ -2520,3 +2524,68 @@ TypeConverter::getContextBoxTypeForCapture(ValueDecl *captured,
   
   return boxType;
 }
+
+static void countNumberOfInnerFields(unsigned &fieldsCount, SILModule &Module,
+                                     SILType Ty) {
+  if (auto *structDecl = Ty.getStructOrBoundGenericStruct()) {
+    assert(structDecl->hasFixedLayout(Module.getSwiftModule(),
+                                      ResilienceExpansion::Minimal) &&
+           " FSO should not be trying to explode resilient (ie address-only) "
+           "types at all");
+    for (auto prop : structDecl->getStoredProperties()) {
```

Comment:
Is this a pointer? Since this is a for loop, can you put in a * to match LLVM style.